var/spack/repos/builtin/packages/riscv-gnu-toolchain/package.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import shlex
+from subprocess import Popen
+
+from spack.package import *
+
+
+class RiscvGnuToolchain(Package):
+    """A cross-compilation tool for RISC-V."""
+
+    homepage = "https://spack-tutorial.readthedocs.io/"
+    git = "https://github.com/riscv-collab/riscv-gnu-toolchain.git"
+
+    phases = ["configure", "build"]
+
+    maintainers = ["wanlinwang"]
+
+    version("develop", branch="master", submodules=True)
+    version("2022.08.08", tag="2022.08.08", submodules=True)
+
+    # Dependencies:
+    depends_on("pkg-config", type=("build", "link"))
+    depends_on("autoconf", type=("build", "link"))
+    depends_on("automake", type=("build", "link"))
+    depends_on("python", type=("build", "link"))
+    depends_on("gawk", type=("build", "link"))
+    depends_on("bison", type=("build", "link"))
+    depends_on("flex@:2.6.1,2.6.4:", type=("build", "link"))
+    depends_on("texinfo", type=("build", "link"))
+    depends_on("patchutils", type=("build", "link"))
+    depends_on("mpc", type=("build", "link"))
+    depends_on("gmp", type=("build", "link"))
+    depends_on("mpfr", type=("build", "link"))
+    depends_on("gcc", type=("build", "link"))
```

Comment:
Is this using `gcc` as a compiler, or to link against libraries from gcc? If it's the former, this is not the right way of doing so.